gradle/testing.gradle

```diff
@@ -20,29 +20,58 @@
 
 if (hasProperty('testEnableCoverage')) {
     apply plugin: 'jacoco'
+    jacoco {
+        toolVersion = jacocoVersion
+    }
     tasks.withType(Test) { task ->
         finalizedBy "jacoco${task.name.capitalize()}Report"
     }
 }
 
+tasks.withType(JacocoReport) { task ->
+    task.reports.xml.required = true
+}
+
+def skipFDB = System.getenv('SKIP_FDB_TESTS') != null && System.getenv('SKIP_FDB_TESTS') == 'true'
+def skipSlow = System.getenv('SKIP_SLOW_TESTS') != null && System.getenv('SKIP_SLOW_TESTS') == 'true'
 test {
-    useJUnit {
+    useJUnitPlatform {
+        if (skipFDB) {
+            excludeTags 'RequiresFDB'
+        }
+        if (skipSlow) {
+            excludeTags 'Slow'
+            systemProperty "junit.jupiter.execution.timeout.default", "2s"
```

Comment:
Note: this timeout is slower, which means that if you run with `SKIP_SLOW_TESTS`, it will fail them faster, but there's nothing that enforces the 2s limit when you don't run with `SKIP_SLOW_TESTS`. I don't know if anyone is currently taking advantage of `SKIP_SLOW_TESTS`.